### ISSUE TITLE
Potential Gradient Graph Issue in compute_policy_loss_clip_cov: log_prob Not Detached

### ISSUE THREAD

**Issue by Ziyang1060** [PRIME-RL/Entropy-Mechanism-of-RL#9]
https://github.com/PRIME-RL/Entropy-Mechanism-of-RL/blob/745b3010bb7b94fec081d3ec072c40404cba7606/verl/trainer/ppo/core_algos.py#L497

In the current implementation of `compute_policy_loss_clip_cov`, I noticed that `log_prob` is not detached from the computation graph during the covariance matrix calculation. While this currently does not affect the final gradient, it could potentially introduce unintended behavior or inefficiencies.

`cov_all = (advantages- verl_F.masked_mean(advantages, response_mask)) * (log_prob- verl_F.masked_mean(log_prob.detach(), response_mask)) `
=> `cov_all = (advantages - verl_F.masked_mean(advantages, response_mask)) * (log_prob.detach() - verl_F.masked_mean(log_prob.detach(), response_mask)) `

Detaching log_prob ensures that unnecessary gradient paths are avoided, which might help with clarity and future-proofing the code.

**Comment by JC-Chen1** [comment]
Thank you for this suggestion! We will update our code.
